drivers/hmis/spec/jobs/auto_exit_job_spec.rb

```diff
@@ -68,6 +68,57 @@
       expect(e1.exit_assessment&.assessment_date).to eq(e1.entry_date)
       expect(e1.exit_assessment&.data_collection_stage).to eq(3)
     end
+
+    let!(:c2) { create :hmis_hud_client, data_source: ds1, user: u1 }
```

Comment:
Id suggest either put this at the top with the rest of a fixtures, or in a `context` that focuses on households